src/game/Chat/Chat.cpp

```diff
@@ -93,22 +93,23 @@ ChatCommand * ChatHandler::getCommandTable()
 
     static ChatCommand partyBotCommandTable[] =
     {
-        { "add",        SEC_ADMINISTRATOR,      false, &ChatHandler::HandlePartyBotAddCommand,         "", nullptr },
-        { "clone",      SEC_ADMINISTRATOR,      false, &ChatHandler::HandlePartyBotCloneCommand,       "", nullptr },
+        { "add",        SEC_PLAYER,      false, &ChatHandler::HandlePartyBotAddCommand,         "", nullptr },
+        { "clone",      SEC_PLAYER,      false, &ChatHandler::HandlePartyBotCloneCommand,       "", nullptr },

```

Comment:
Why are you changing the permission from `SEC_ADMINISTRATOR` to `SEC_PLAYER` here? That doesn't seem right.